src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -74,6 +74,22 @@
  * <pre>
  * &lt;module name="BooleanExpressionComplexity"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <div class="wrapper">
+ * <pre>
```

Comment:
Javadocs should only have `pre` tag, not `div`.

Reply:
done